src/Http/Controllers/MetadataController.php

```diff
@@ -19,8 +19,12 @@ public function index()
             \Barryvdh\Debugbar\Facade::disable();
         }
 
-        return response(view('samlidp::metadata'), 200, [
-            'Content-Type' => 'application/xml'
+        $cert = Storage::disk('samlidp')->get('cert.pem');
+        $cleanedUpCert = preg_replace('/^\W+\w+\s+\w+\W+\s(.*)\s+\W+.*$/s', '$1', $cert);
+        $cleanedUpCert = str_replace(PHP_EOL, "", $cleanedUpCert);
```

Comment:
@agmadt I think we can reuse `$cert` here. It's not a huge memory saver, but it makes sense. Then of course `$cert` will need to be compacted and updated in the view.